client/lib/plans/features-list.js

```diff
@@ -33,6 +33,7 @@ export const FEATURES_LIST = {
 					),
 				},
 			} ),
+		getTrackingLabel: () => 'All Free features',
```

Comment:
I don't see the rationale of having to add these new `getTrackingLabel` functions to the features list. Instead, this should be coming from the same place where the link text is defined. Otherwise, if we change the link text, we also have to change this one, but we can easily forget about that, and that will add a lot of confusion.

Reply:
@tyxla There are two reasons why this is implemented in such a way:

1. The `getTitle` function returns a `i18n.translate` function. It, in turn, consists not only of just a plain string but also components (`<a />` in this case). I found it really hard to use it as a string `label` in the Tracks event.
2. Even if the title was just a simple string without any other components inside, we still would run into an issue of having multiple events recorded in Tracks that would virtually be the same but would have different `label`s depending on locale.

For these reasons [I suggested having a more general `label`](https://github.com/Automattic/wp-calypso/pull/37442#discussion_r344213308).

I'm going to reimplement what we have here so that we make use of slugs instead of titles/labels so those issues should go away.

Reply:
Using slugs instead of the actual labels makes a whole lot of sense, thanks! 😉